src/pages/FeedbackModal.jsx

```diff
@@ -52,7 +75,12 @@ export const FeedbackModal = ({ visible, setVisible }) => {
       <FeedbackWrapper>
         <FeedbackList>
           {reviews.map((review) => (
-            <Review {...review} key={review.id} style={{ border: 'none', padding: 0 }} />
+            <>
+              <EditWrapper onClick={() => handleEditFeedbackClick(review.id)}>
```

Comment:
You can use OpacityButton componet from buttons/ folder, please check how it works, then you will have opacity effect